packages/salesforcedx-vscode-core/package.json

```diff
@@ -749,6 +750,11 @@
           "type": "boolean",
           "default": false,
           "description": "%conflict_detection_enabled_description%"
+        },
+        "salesforcedx-vscode-core.tooling-deploys": {
```

Comment:
We should rename this to something more generic that we can use for the rest of the changes we'll be doing so that everything fits under one setting